From db04e4469bad9fc90894524d0f4f5cbf1a803300 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Thu, 21 Jul 2016 19:11:39 +0000
Subject: [PATCH] TOOLS-1493 gerrit ignores modifier keys in firefox; eats some
 keystrokes Reviewed by: Dave Pacheco <dap@joyent.com> Approved by: Dave
 Pacheco <dap@joyent.com>

---
 images/appserver/Dockerfile | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/images/appserver/Dockerfile b/images/appserver/Dockerfile
index 57005f7..9e6bc32 100644
--- a/images/appserver/Dockerfile
+++ b/images/appserver/Dockerfile
@@ -14,13 +14,16 @@ ENV GERRIT_RELEASEURL         https://gerrit-releases.storage.googleapis.com
 #ENV GERRIT_WAR_URL            ${GERRIT_RELEASEURL}/gerrit-${GERRIT_VERSION}.war
 
 #
-# Work around "https://gerrit-review.googlesource.com/#/c/74830/2" by using
-# a custom build of 2.12.2 with the upstream fix for this issue included
-# as well.  Once a release that includes this fix is available, we should
-# switch back to using official Gerrit release artefacts.
+# Work around several issues by using a custom build of 2.12.2 with the
+# upstream fixes included.  Once a release that includes these fixes is
+# available, we should switch back to using official Gerrit release artefacts.
+#
+# Current included workarounds:
+#     https://gerrit-review.googlesource.com/#/c/74830
+#     https://gerrit-review.googlesource.com/#/c/75987
 #
 ENV GERRIT_OVERRIDEURL        https://us-east.manta.joyent.com/Joyent_Dev/public/gerrit/custom
-ENV GERRIT_WAR_URL            ${GERRIT_OVERRIDEURL}/gerrit-2.12.2-joyent1.war
+ENV GERRIT_WAR_URL            ${GERRIT_OVERRIDEURL}/gerrit-2.12.2-joyent2.war
 
 ENV PLUGIN_VERSION            stable-2.12
 ENV GERRITFORGE_URL           https://gerrit-ci.gerritforge.com
-- 
2.21.0

